defaults/main.yml

```diff
@@ -60,6 +60,9 @@ datadog_agent_version: ""
 # Default Package name for APT and RPM installs - can override in playbook for IOT Agent
 datadog_agent_flavor: "datadog-agent"
 
+# Default Package name for APM Library Injection APT and RPM installs - can override in playbook for IOT Agent
```

Comment:
Can you explain what you mean by "can override in playbook for IOT Agent"?

Reply:
I will leave this one to @jonbodner